components/Layout.tsx

```diff
@@ -1,49 +1,35 @@
-import Link from 'next/link';
-import Head from 'next/head';
+import { Button } from '@material-ui/core';
+import styles from '../styles/Home.module.css';
 
 type Props = {
   title?: string;
 };
 
 const Layout: React.FunctionComponent<Props> = ({
   children,
-  title = 'This is the default title',
+  title = 'NBJC',
 }) => (
   <div>
-    <Head>
-      <title>{title}</title>
-      <meta charSet="utf-8" />
-      <meta name="viewport" content="initial-scale=1.0, width=device-width" />
-    </Head>
-    <header>
-      <nav>
-        <Link href="/">
-          <a>Home</a>
-        </Link>{' '}
-        |{' '}
-        <Link href="/about">
-          <a>About</a>
-        </Link>{' '}
-        |{' '}
-        <Link href="/users">
-          <a>Users List</a>
-        </Link>{' '}
-        |{' '}
-        <Link href="/moderator">
-          <a>Moderator Dash</a>
-        </Link>{' '}
-        |{' '}
-        <Link href="/orgs">
-          <a>Orgs List</a>
-        </Link>{' '}
-        | <a href="/api/users">Users API</a>
-      </nav>
-    </header>
+    <div className={styles.pageTitle}>
```

Comment:
we want to keep the Head component because that changes the title on the tab bar